config/index.js

```diff
@@ -209,21 +245,25 @@ function get(varName) {
   return fs.readFileSync(process.env[fileVarName], 'utf8').trim();
 }
 
-// Ensures all properties of the configuration are valid.
-function validate(config) {
-  if (!_.isInteger(config.bcryptCost) || config.bcryptCost < 1) {
-    throw new Error(`Unsupported bcrypt cost "${config.bcryptCost}" (type ${typeof(config.bcryptCost)}); must be an integer greater than or equal to 1`);
-  } else if (!_.isBoolean(config.cors)) {
-    throw new Error(`Unsupported CORS value "${config.cors}" (type ${typeof(config.cors)}); must be a boolean`);
-  } else if (!_.isString(config.db) || !config.db.match(/^postgres:\/\//)) {
-    throw new Error(`Unsupported database URL "${config.db}" (type ${typeof(config.db)}); must be a string starting with "postgres://"`);
-  } else if (!_.includes(SUPPORTED_ENVIRONMENTS, config.env)) {
-    throw new Error(`Unsupported environment "${JSON.stringify(config.env)}"; must be one of: ${SUPPORTED_ENVIRONMENTS.join(', ')}`);
-  } else if (!_.isString(config.logLevel) || !_.includes(SUPPORTED_LOG_LEVELS, config.logLevel.toUpperCase())) {
-    throw new Error(`Unsupported log level "${config.logLevel}" (type ${typeof(config.logLevel)}); must be one of: ${SUPPORTED_LOG_LEVELS.join(', ')}`);
-  } else if (!_.isInteger(config.port) || config.port < 1 || config.port > 65535) {
-    throw new Error(`Unsupported port number "${config.port}" (type ${typeof(config.port)}); must be an integer between 1 and 65535`);
-  } else if (!_.isString(config.sessionSecret) || config.sessionSecret == 'changeme') {
-    throw new Error(`Unsupported session secret "${config.sessionSecret}" (type ${typeof(config.sessionSecret)}); must be a string different than "changeme"`);
+/**
+ * Ensures all properties of the configuration are valid.
+ *
+ * @param {Object} conf - The configuration object to validate.
+ */
+function validate(conf) {
+  if (!_.isInteger(conf.bcryptCost) || conf.bcryptCost < 1) {
+    throw new Error(`Unsupported bcrypt cost "${conf.bcryptCost}" (type ${typeof conf.bcryptCost}); must be an integer greater than or equal to 1`);
+  } else if (typeof conf.cors !== 'boolean') {
```

Comment:
Yeah... I really have mixed feelings about using some lodash type-checking function and using `typeof` for some others...
I think I would prefer to enforce the user of the `util` ones, as it's a Node native package, but I'm still not completely sure.